easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422' 
```

Comment:
yeah, basically do something like:

```python
version = '20150415'
commit = '4ef0705'
...
source_urls = ['https://github.com/pierrepeterlongo/commet/archive']
sources = ['%s.tar.gz' % commit]
```

note that I'm using the date that corresponds to that commit